docs/flow/reference/cli/run.md

```diff
@@ -6,3 +6,29 @@ weight: 100
 ---
 
 # `agent run` command
+
+The `agent run` command runs Grafana Agent Flow in the foreground until an
```

Comment:
Do we want to mention the `EXPERIMENTAL_ENABLE_FLOW` flag here, or leave it just in the code for now?

Reply:
All of the Flow docs only apply when `EXPERIMENTAL_ENABLE_FLOW` is set so I think we'll mention it somewhere higher-level than this.